sdk/servicebus/service-bus/src/track2/serviceBusReceiverClient.ts

```diff
@@ -0,0 +1,526 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
```

Comment:
Not sure why this doesn't throw a compiler error but I don't see "Long" imported here. (That's probably why the api review file shows a `Long_2` somewhere.

Reply:
Added and I'm also not sure why that wasn't an error.